kopi/assetlist.json

```diff
@@ -0,0 +1,537 @@
+      "name": "USDC",
+      "display": "USDC",
+      "symbol": "USDC",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdc.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdc.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "noble",
+            "base_denom": "uusdc",
+            "channel_id": "channel-109"
+          },
+          "chain": {
+            "channel_id": "channel-0",
+            "path": "transfer/channel-109/uusdc"
+          }
+        }
+      ]
+    },
+    {
+      "description": "USDT from injective Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/D8A36AE90F20FE4843A8D249B1BCF0CCDDE35C4B605C8DED57BED20C639162D0",
+          "exponent": 0
+        },
+        {
+          "denom": "USDT.inj",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/D8A36AE90F20FE4843A8D249B1BCF0CCDDE35C4B605C8DED57BED20C639162D0",
+      "name": "USDT.inj",
+      "display": "USDT.inj",
+      "symbol": "USDT.inj",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "ibc/4ABBEF4C8926DDDB320AE5188CFD63267ABBCEFC0583E4AE05D6E5AA2401DDAB",
+            "channel_id": "channel-376"
+          },
+          "chain": {
+            "channel_id": "channel-1",
+            "path": "transfer/channel-376/ibc/4ABBEF4C8926DDDB320AE5188CFD63267ABBCEFC0583E4AE05D6E5AA2401DDAB"
+          }
+        }
+      ]
+    },
+    {
+      "description": "injective from injective Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/DE63D8AC34B752FB7D4CAA7594145EDE1C9FC256AC6D4043D0F12310EB8FC255",
+          "exponent": 0
+        },
+        {
+          "denom": "INJ",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/DE63D8AC34B752FB7D4CAA7594145EDE1C9FC256AC6D4043D0F12310EB8FC255",
+      "name": "Injective",
+      "display": "Inj",
```

Comment:
This display unit must be defined, but it's not found anywhere in the denom_units array. I believe you're looking for "INJ", since that is defined as a unit, and it exponent 18